docs/streams/developer-guide/testing.html

```diff
@@ -275,21 +275,21 @@ <h2>
             <b>Construction</b>
             <p>
                 To begin with, instantiate your processor and initialize it with the mock context:
-            <pre class="line-numbers"><code class="language-text">final Processor processorUnderTest = ...;
+            <pre class="line-numbers"><code class="language-java">final Processor processorUnderTest = ...;
 final MockProcessorContext context = new MockProcessorContext();
 processorUnderTest.init(context);</code></pre>
             If you need to pass configuration to your processor or set the default serdes, you can create the mock with
             config:
-            <pre class="line-numbers"><code class="language-text">final Properties props = new Properties();
+            <pre class="line-numbers"><code class="language-java">final Properties props = new Properties();
 props.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.String().getClass());
 props.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.Long().getClass());
-props.put("some.other.config", "some config value");
+props.put(&quot;some.other.config&quot;, &quot;some config value&quot;);
 final MockProcessorContext context = new MockProcessorContext(props);</code></pre>
             </p>
             <b>Captured data</b>
             <p>
                 The mock will capture any values that your processor forwards. You can make assertions on them:
-            <pre class="line-numbers"><code class="language-text">processorUnderTest.process("key", "value");
+            <pre class="line-numbers"><code class="language-java">processorUnderTest.process("key", "value");
```

Comment:
On line 304 and line 306 are two code snippets that slipped through.

Reply:
Done!